tests/smoke_tests.py

```
@@ -0,0 +1,111 @@
+    print("✅ Configuration values accessible")
+    return True
+
+
+def test_django_settings_imports():
+    print("Testing Django settings imports...")
+    from importlib import import_module
+    try:
+        import_module("endo_api.settings_dev")
+        import_module("endo_api.settings_prod")
+        print("✅ Django settings modules import successfully")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)